### PR TITLE
docs(aio): more content fixes

### PR DESCRIPTION
**Pull request by Foxandxss**
Some extra authoring changes.

### CODE REVIEW

File: aio/content/guide/webpack.md [hunk 1]
```diff
@@ -137,22 +137,17 @@ It's preferable to separate the volatile application app code from comparatively
 Change the configuration so that it has two entry points, `main.ts` and `vendor.ts`:
 
 
-<div class='code-example'>
```
Comment (gkalpak) [inline aio/content/guide/webpack.md]: There is one more unnecessary `<div class='code-example'>` on line 113.

**Review by gkalpak — COMMENTED** [review]
The ToC for `ajs-quick-reference` has too many entries. It might be better to only list the AngularJS things (e.g. `ng-class` and not `ngClass`). Most people visiting the page will know the AngularJS thing and look for the corresponding Angular thing (not the other way around).

File: aio/content/guide/npm-packages.md [hunk 2]
```diff
@@ -194,7 +194,7 @@ with excellent support for Angular apps that use routing.
 ***[typescript](https://www.npmjs.com/package/typescript)***:
 the TypeScript language server, including the *tsc* TypeScript compiler.
 
-***@types/\****: TypeScript definition files.
+***@types/\* ***: TypeScript definition files.
```
Comment (Foxandxss) [inline aio/content/guide/npm-packages.md]: if I don't put the space, it renders wrong, one asterisc before and after.

File: aio/content/guide/webpack.md (hunk 1 above)
Comment (Foxandxss) [inline aio/content/guide/webpack.md]: Fixed.

**Comment by Foxandxss** [comment]
I put the `hideCopy` in the quick reference guide (with a fix for dgeni) and fixed the other concern.

File: aio/tools/transforms/helpers/utils.js [hunk 3]
```diff
@@ -92,6 +92,6 @@ module.exports = {
   },
 
   renderAttributes(attrMap) {
-    return Object.keys(attrMap).map(key => ` ${key}="${attrMap[key].replace(/"/g, '&quot;')}"`).join('');
+    return Object.keys(attrMap).map(key => (attrMap[key] === true) ? ` ${key}` : ` ${key}="${attrMap[key].replace(/"/g, '&quot;')}"`).join('');
```
Comment (gkalpak) [inline aio/tools/transforms/helpers/utils.js]: OK, I see it comes from `parseAttributes()` above 😕

**Review by gkalpak — APPROVED** [review]

File: aio/tools/transforms/helpers/utils.js (hunk 3 above)
Comment (Foxandxss) [inline aio/tools/transforms/helpers/utils.js]: (The fix was made by Pete, I just copied it here :P)

Comment (petebacondarwin) [inline aio/tools/transforms/helpers/utils.js]: I'll knock up a PR with tests for this change later today.

**Pull request merged**